### PR TITLE
Add Coqui TTS functionality

### PR DESCRIPTION
**Pull request by howardbaek**
- Since ari doesn't play well when Coqui TTS is output as `mp3`, I made sure to set the `output_format` of `text2speech::tts()` to be `wav` instead of `mp3`. 
- Changed Remotes field in DESCRIPTION to `jhudsl/text2speech`

### CODE REVIEW

File: R/ari_spin.R [hunk 1]
```diff
@@ -102,51 +103,59 @@ ari_spin <- function(images, paragraphs,
...
+    if (service == "coqui") {
+      args$output_format <- "wav"
+    }
+
     wav <- do.call(text2speech::tts, args = args)
     wav <- reduce(wav$wav, bind)
     wav <- pad_wav(wav, duration = duration[i])
     ideal_duration[i] <- length(wav@left) / wav@samp.rate
-    wavs[[i]] <- wav
+    wave_objects[[i]] <- wav
```
Comment (cansavvy) [inline R/ari_spin.R:139]: Why is this different?

**Review by cansavvy — CHANGES_REQUESTED** [review]
Seems reasonable. Just two things:

1) Cam you document everything you know at this time about why mp3 isn't working for ari with coqui? What error messages or behaviors are you seeing? Putting this in a GitHub issue. We can try to address this later. 
2) Then for now, let's just print out a warning message to the user that mp3 and coqui is not currently supported. This way if they set their argument to mp3 they wouldn't be confused or surprised as to why they get a wav file.

File: R/ari_spin.R (hunk 1 above)
Comment (howardbaek) [reply R/ari_spin.R:139]: I changed the variable name to be something more descriptive

**Comment by howardbaek** [comment]
Responding to @cansavvy,

I was running through the source code of `ari_spin()`, which creates Wave objects in [these lines](https://github.com/jhudsl/ari/blob/main/R/ari_spin.R#L128-L132). Then, it calls `ari_stitch()`, which calls [`writeWave`](https://github.com/jhudsl/ari/blob/main/R/ari_stitch.R#L145) which writes WAV audio files when you input Wave objects. 
If I create these Wave objects with coqui when output format is mp3, then write WAV audio files, I get a broken,scratchy radio sound. But, if I create these Wave objects with coqui when output format is WAV, I get normal sounding audio.

I asked on Coqui's Discord about the coqui's output format. The response that I got is the native output option with coqui is WAV and if I choose to output with MP3, I will get a file with an mp3 extension that is wav-encoded. So, they recommended I make the WAV file with coqui.

File: R/ari_spin.R [hunk 2]
```diff
@@ -102,51 +103,59 @@ ari_spin <- function(images, paragraphs,
...
-
+  # Iterate through arguments used in tts()
   for (i in par_along) {
     args <- tts_args
     args$text <- paragraphs[i]
     args$voice <- voice
     args$service <- service
     args$bind_audio <- TRUE
+    # coqui+ari doesn't work with mp3
+    if (service == "coqui") {
```
[the remark this answers is not in the document]
Comment (howardbaek) [reply R/ari_spin.R:131]: Good idea

**Comment by howardbaek** [comment]
@cansavvy  I've just added a warning message to `ari_spin()` and ran `devtools::document()`. This changed the .Rd files of other functions, but these changes are all stylistic changes (spacing, arrows as assignment operators, etc)

File: R/ari_spin.R [hunk 3]
```diff
@@ -130,8 +131,8 @@ ari_spin <- function(images, paragraphs,
     # coqui+ari doesn't work with mp3
     if (service == "coqui") {
       args$output_format <- "wav"
+      cli::cli_alert_info("Coqui TTS does not support MP3 format and will produce a WAV audio output.")
```
Comment (cansavvy) [inline R/ari_spin.R:134]: Was this already a package that was dependency in here? Also how is this different than using `warning()` or `message()`? I've never heard of this package so I'm just curious.

Comment (howardbaek) [reply R/ari_spin.R:134]: I need to add cli as a dependency in the docs! Also, this isn't much different from a `message()` except it makes prettier messages, like https://github.com/r-lib/cli#short-alert-messages

**Comment by howardbaek** [comment]
> > I was running through the source code of `ari_spin()`, which creates Wave objects in [these lines](https://github.com/jhudsl/ari/blob/main/R/ari_spin.R#L128-L132). Then, it calls `ari_stitch()`, which calls [`writeWave`](https://github.com/jhudsl/ari/blob/main/R/ari_stitch.R#L145) which writes WAV audio files when you input Wave objects. If I create these Wave objects with coqui when output format is mp3, then write WAV audio files, I get a broken,scratchy radio sound. But, if I create these Wave objects with coqui when output format is WAV, I get normal sounding audio.
> 
> Very basic question: Does ari only work with wav objects for the other text2speech services as well?



I tested running ari with WAV as an output format and everything ran smoothly for Amazon/Google. I lost my free trial access to Microsoft Azure, so haven't been able to test Microsoft.

File: DESCRIPTION [hunk 4]
```diff
@@ -1,46 +1,49 @@
-Package: ari
 Type: Package
+Package: ari
 Title: Automated R Instructor
 Version: 0.4.1
 Authors@R: c(
-    person("Sean", "Kross", email = "sean@seankross.com", role = c("aut", "cre")),
+    person("Sean", "Kross", , "sean@seankross.com", role = c("aut", "cre")),
```
Comment (cansavvy) [inline DESCRIPTION:6]: Looks like there’s an extra comma here

**Comment by cansavvy** [comment]
We should look into that dev Ubuntu fail and try to fix it but otherwise this seems good to me.

**Review by cansavvy — APPROVED** [review]

**Pull request merged**